native/android/build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.3"
+        buildToolsVersion = "31.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 29
-        targetSdkVersion = 29
```

Comment:
I'am not sure if we want to increase the target sdk version here. Did you find out why uses androidx.browser:browser:1.4.0-alpha01 instead of 1.3.0? Maybe setting it to use 1.3.0 will also fix the build.

Reply:
sth like https://stackoverflow.com/a/69249047

Reply:
I finally found the culprit [react-native-inapp-browser](https://github.com/proyecto26/react-native-inappbrowser) fetches the newest version of androidx.browser:browser. To fix this you can add `androidXBrowser = "1.3.0"` in the android/build.gradle